go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	sigs.k8s.io/gateway-api v0.4.3
 )
 
-replace github.com/l7mp/stunner => ../stunner
+replace github.com/l7mp/stunner => github.com/l7mp/stunner v0.10.9-0.20220928203107-a2d718d90804
```

Comment:
I'm so sorry, this was my fault, forgot to tag a release. STUNner v.0.10.9 is now tagged, could you please update the dependency accordingly?

Reply:
done in [5d90990](https://github.com/l7mp/stunner-gateway-operator/pull/6/commits/5d909909f32b8b8703710555d469c7a6daf98ad9)